address.go

```diff
@@ -75,6 +76,11 @@ type createAddressRequest struct {
 	Address *Address `json:"address,omitempty"`
 }
 
+type AddressVerifyResponse struct {
```

Comment:
Please double check the documentation surrounding address verification. I don't believe this is the correct way to set this up. If you look at the recorded cassette, you'll notice that `message` does not appear here at all.

Reply:
I was using Java for reference: https://github.com/EasyPost/easypost-java/blob/master/src/main/java/com/easypost/model/Address.java#L144-L157

The API is going to return a `message` and `Address`, we set the `message` from API call to `Address` if it's not null. Therefore, we need to create another struct that has two attributes `message` and `Address`.

Reply:
Can you show me where the API returned a message and address? I don't see that in the recorded cassette.

Reply:
Yes, but we are not going to use the `AddressVerificationFieldError` struct because it's for something else and does not have the `Address` attribute.

Reply:
After internal discussion, we found out that this endpoint returns the verified address in a nested dictionary, we don't need the `message` attribute in this struct.